src/assets/config_files/usecases.json

```diff
@@ -47,6 +47,26 @@
           }
         ]
       },
+      {
+        "title": "Predict protein interaction surface with the ARDOCK server",
+        "description": "In silico analysis of your protein surface using molecular probes to reveal potential interaction sites",
+        "experience": ["all"],
+        "maturity": ["beta"],
+        "disabled": false,
```

Comment:
You can add also some Collaborators if you want (it's optional)
Like https://github.com/antonelepfl/usecases/pull/214/files#diff-792ef00d694f11cf071142152aecd528R160